build.sh

```diff
@@ -66,6 +66,7 @@ Usage: $0 <options>
 
   Environment variables:
     USE_AVX2                    If the CPU does not support AVX2 instruction set, please set USE_AVX2=0. Default is ON.
+    USE_NATIVE                  Support -march=native compile to speed up the different CPU in cloud, if USE_NATIVE is ON, the USE_AVX2 auto turn off. Default is OFF.
```

Comment:
it seems not related to cloud mode actually, every one who build from source can use this option.